sdk/core/azure-core/src/main/java/com/azure/core/implementation/SwaggerMethodParser.java

```diff
@@ -109,7 +111,8 @@
             requiredAnnotationOptions.add(Delete.class);
             requiredAnnotationOptions.add(Post.class);
             requiredAnnotationOptions.add(Patch.class);
-            throw new MissingRequiredAnnotationException(requiredAnnotationOptions, swaggerMethod);
+            throw logger.logExceptionAsError(new MissingRequiredAnnotationException(requiredAnnotationOptions,
```

Comment:
Same comment as the other swagger class logging

Reply:
Revert it back.